docs/api-spec-generated/security.md

```diff
@@ -23,5 +23,41 @@ scopes in the future such as limiting access to particular event types and strea
 |nakadi.event_stream.write|Grants access for applications to submit Events.|
 |nakadi.event_stream.read|Grants access for consuming Event streams.|
 
-
-
+<a name="event type"></a>
+### Event Type
+#### Write and read scopes
+
+Nakadi has possibility to check OAuth2 Scopes (scopes) of publisher and consumer to grant access
+write and read events only for trusted users.
```

Comment:
maybe replace with "trusted services" or "trusted applications" ? Because those are not the real users.